mastering-plone/behaviors_2.rst

```diff
@@ -327,7 +343,8 @@ Exercises
 Exercise 1
 ++++++++++
 
-Refactor the voting behavior so that it uses `BTrees` instead of `PersistentDict` and `PersistentList`. Use `OOBTree` to replace `PersistentDict` and `OIBTree` to replace `PersistentList`.
+Refactor the voting behavior so that it uses `BTrees` instead of `PersistentDict` and `PersistentList`.
+Use `OOBTree` to replace `PersistentDict` and `OIBTree` to replace `PersistentList`.
```

Comment:
We want double backticks instead for these inline literals. Single backticks in markdown, not reST, are inline literals.

Reply:
Well, I just added line breaks to increase plain text readability. It seems the whole training needs a PR only about style. We may want to do this separate, what do you think?

Reply:
Big changes all less likely to happen. Small changes are more doable. Enforcing double backticks might be done with a regex:

```
Find:
(\s)`(.*?)`(\s)

Replace:
\1``\2``\3
```

...is a start, but would need to be carefully audited before committing.